### ISSUE TITLE
Uncaught Error: EACCES: permission denied, scandir '$ATOM_HOME/packages/terminal-plus'

### ISSUE THREAD

**Issue by jameswilliamknight** [atom/settings-view#988]
[Enter steps to reproduce:]

1. install 'terminal-plus' via `apm install LarsKumbier/terminal-plus`
2. Go to **Settings** > **Installed Packages** and click on the 'terminal-plus' package.

**Atom**: 1.19.5 x64
**Electron**: 1.6.9
**OS**: Ubuntu 17.04
**Thrown From**: [settings-view](https://github.com/atom/settings-view) package 0.250.0

### Stack Trace

Uncaught Error: EACCES: permission denied, scandir '/home/james/.atom/packages/terminal-plus'

```
At fs.js:871

Error: EACCES: permission denied, scandir '/home/james/.atom/packages/terminal-plus'
    at Object.fs.readdirSync (fs.js:871:18)
    at Object.fs.readdirSync (ELECTRON_ASAR.js:576:28)
    at Proxy.listSync (/usr/share/atom/resources/app/node_modules/fs-plus/lib/fs-plus.js:190:30)
    at PackageDetailView.updateFileButtons (/usr/share/atom/resources/app/node_modules/settings-view/lib/package-detail-view.js:588:51)
    at PackageDetailView.updateInstalledState (/usr/share/atom/resources/app/node_modules/settings-view/lib/package-detail-view.js:484:18)
    at PackageDetailView.populate (/usr/share/atom/resources/app/node_modules/settings-view/lib/package-detail-view.js:454:18)
    at PackageDetailView.completeInitialization (/usr/share/atom/resources/app/node_modules/settings-view/lib/package-detail-view.js:214:18)
    at PackageDetailView.loadPackage (/usr/share/atom/resources/app/node_modules/settings-view/lib/package-detail-view.js:225:20)
    at new PackageDetailView (/usr/share/atom/resources/app/node_modules/settings-view/lib/package-detail-view.js:100:16)
    at callback (/usr/share/atom/resources/app/node_modules/settings-view/lib/settings-view.js:356:26)
    at SettingsView.getOrCreatePanel (/usr/share/atom/resources/app/node_modules/settings-view/lib/settings-view.js:361:25)
    at SettingsView.showPanel (/usr/share/atom/resources/app/node_modules/settings-view/lib/settings-view.js:419:30)
    at HTMLDivElement.clickHandler (/usr/share/atom/resources/app/node_modules/settings-view/lib/package-card.js:285:39)
```

### Commands

```
  3x -3:52.6.0 intentions:highlight (input.hidden-input)
```

### Non-Core Packages

```
atom-beautify 0.30.5 
busy-signal 1.4.3 
file-icons 2.1.11 
intentions 1.1.5 
linter 2.2.0 
linter-jshint 3.1.5 
linter-ui-default 1.6.8 
markdown-preview-plus 2.4.10 
terminal-plus undefined 
```

**Comment by rsese** [comment]
Sorry for the delay @jameswilliamknight, are you still seeing this error?  If so, can you share the permissions on the directory?

```
ls -ld /home/james/.atom/packages/terminal-plus
```

We just looked at a similar report at https://github.com/atom/atom/issues/15952 and found it was a permissions issue on the package directory.

**Comment by jameswilliamknight** [comment]
I just tried to install again to give you the `ls` permission output. Having some trouble doing it through
`sudo apm install LarsKumbier/terminal-plus`

but installing through the package manager / atom gui works now (previously didn't). 

this is probably irrelevant now:
`drwxrwxr-x 9 james james 4096 Oct 24 19:02 /home/james/.atom/packages/terminal-plus`

but now I'm at the point where it's installed but i bring up the terminal and it's black. I'll troubleshoot. You could probably close this issue. Something funky with the `sudo/github` install method, my guess is installing it elevated and atom trying to access it not elevated.

**Comment by rsese** [comment]
ahhh yeah I'm guessing the problem is using `sudo` with the package install like you alluded to.  I think if you just `apm install LarsKumbier/terminal-plus` it should work.

About the black screen with terminal plus, I ran into that too a while back and it seems like it hasn't been updated in a while:

https://discuss.atom.io/t/terminal-plus/48225/2

I also gave [termination](https://atom.io/packages/termination) a try and that's what I use now.
